src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationForm.test.tsx

```diff
@@ -194,6 +194,11 @@ describe('PublicationForm', () => {
       'Test title',
     );
 
+    await userEvent.type(
+      screen.getByLabelText('Publication summary'),
+      'Test summary',
+    );
+
```

Comment:
I think you could also add a new test `shows validation error when there is no summary` given there are variations like `shows validation error when there is no title` etc.